flink-python/pyflink/pyflink_gateway_server.py

```diff
@@ -71,7 +71,7 @@ def read_from_config(key, default_value, flink_conf_file):
 def find_java_executable():
     java_executable = "java.exe" if on_windows() else "java"
     flink_home = _find_flink_home()
-    flink_conf_file = os.path.join(flink_home, "conf", "flink-conf.yaml")
+    flink_conf_file = os.path.join(flink_home, "conf", "config.yaml")
```

Comment:
We need a fallback to `flink-conf.yaml` for the older versions. Otherwise, this LGTM.

Reply:
Yes, I've decided to not do this change, but use an old `flink-conf.yaml` file and copy that to all locations where a `config.yaml` exists. Since that's the backwards guarantee for all Flink 1.* releases, I think this is a safe way for us to run all the Python tests